plugins/org.eclipse.n4js/src/org/eclipse/n4js/N4JS.xtext

```diff
@@ -339,11 +340,15 @@ FormalParameter <Yield>:
 
 fragment BindingElementFragment <Yield>*:
 	( => bindingPattern=BindingPattern<Yield>
-	| annotations+=Annotation* variadic?='...'? name=BindingIdentifier<Yield> ColonSepDeclaredTypeRef?
+	| annotations+=Annotation* BogusTypeRefFragment? variadic?='...'? name=BindingIdentifier<Yield> ColonSepDeclaredTypeRef?
 	)
 	(hasInitializerAssignment?='=' initializer=AssignmentExpression<In=true, Yield>?)?
 ;
 
+fragment BogusTypeRefFragment*:
+	bogusTypeRef=TypeRefWithModifiers
+;
```

Comment:
It seems this slipped back in (also in `N4JS.xcore` in class `TypedElement`).

Reply:
Yes, Thanks!